### ISSUE TITLE
Parenthesis inside tokens are not parsable

### ISSUE THREAD

**Issue by yochail** [yzhangcs/parser#120]
Hi, Thanks for all the good work!

I'm not working with a stanza tokenizer, but with a customized spacy tokenizer, but I think that like my tokenizer,
for the input "hello world :)" stanza will return the tokens ["hello" "world" ":)"] which will throw an exception inside NLTK dataset constructor.

This [issue ](https://github.com/yzhangcs/parser/issues/109) was partly fixed, and [this fix](https://github.com/yzhangcs/parser/issues/109#issuecomment-1209444998_) is working for single Tokens like '(' or ')', but for tokens with more than a single parenthesis, mainly emojis like ':)', '(8', etc. the issue still exists and causes an error.

**Comment by yzhangcs** [comment]
@yochail Hi, could you provide me the constructed examples that will raise errors? (please take care to use the latest commits)
I've tried this, and the outputs are ok:
```py
>>> Parser.load('crf-con-en').predict(["hello world :)"], lang='en', verbose=False)[0].pretty_print()
       TOP        
        |          
        NP        
   _____|_____     
  _     _     _   
  |     |     |    
hello world :-RRB-
```

**Comment by yochail** [comment]
@yzhangcs Thanks! i see it was already fixed in commit https://github.com/yzhangcs/parser/commit/fe5c20395950187989955444a72c8a0e021ee389
but I didn't get the fix since I'm using the latest release 1.1.4
Are there any plans to release a new stable version in the near future?
